snc.sh

```diff
@@ -10,19 +10,30 @@ YQ=${YQ:-yq}
 OPENSHIFT_INSTALL=${OPENSHIFT_INSTALL:-./openshift-install}
 CRC_VM_NAME=${CRC_VM_NAME:-crc}
 BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
-QUAY_REGISTRY=${QUAY_REGISTRY:-quay.io/openshift-release-dev/ocp-release}
+MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/clients/ocp}
```

Comment:
@wking So is this correct assumption that `mirror.openshift.com` is source of truth when it comes to OCP releases? Recently when 4.3 rc bits happened those were not appeared on mirror.openshift.com atleast for a day or two. If you says this is way forward and every bits should be present here then we can merge it.

Reply:
> Recently when 4.3 rc bits happened those were not appeared on mirror.openshift.com at least for a day or two...

They're supposed to get pushed up around when the release is tagged into the candidate channel, with `latest*` symlinks bumped around when a release is tagged into fast/stable.  If that's not what's happening, talk to ART.